doc/cookbook/source/examples/classifier/random_forest.rst

```diff
@@ -0,0 +1,41 @@
+=============
+Random Forest
+=============
+
+Random Forest is an ensemble learning method which implements multiple decision trees during training. It predicts by using a combination rule on the outputs of individual decision trees.
+
+See :cite:`Breiman2001` for a detailed introduction.
+
+-------
+Example
+-------
+
+CDenseFeatures (here 64 bit floats aka RealFeatures) and :sgclass:`CMulticlassLabels` are created from training and test data file
```

Comment:
AFAIK, the current implementation cannot do that. 
The Random forest interface can, but the CartTree inside assumes float64_t.
Should definitely be changed, see email I wrote. 
We can (and should) update this cookbook once Shogun supports these things